.gitignore

```diff
@@ -63,3 +63,7 @@ ipch/
 CMakeLists.txt
 compile_commands.json
 /.gitmodules
+
+# static code analysis
+/.scannerwork
+/compile_commands.json
```

Comment:
Isn't the compile_commands.json already excluded by line 64?

Reply:
Yeah, I guess it is.